inkcpp_compiler/include/compiler.h

```diff
@@ -2,7 +2,7 @@
 
 #include "config.h"
 #ifdef INK_EXPOSE_JSON
-#include "json.hpp"
+#include "../json.hpp"
```

Comment:
In Linux the compiler is quite precise when you use a relative path `#include "header"`, the `json.hpp` is one directory about `compiler.h`.